### ISSUE TITLE
Modify button copy/style on filter modal

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#827]
In order to make it consistent with the rest of the design, the styling of the 'Apply filters' button should be rounded (and the color modified to be consistent with other buttons). Also, the copy should be updated to be 'Apply filter'.

Design is viewable in [Figma](https://www.figma.com/file/wAhoXVmHRIJ2TyBKFVq2JY/Desktop-and-Mobile-Experiences?node-id=3540%3A13497) (02.2).

![filter_modal_button](https://user-images.githubusercontent.com/394908/145234877-91296379-c145-4f98-8a54-e8d08ab133df.png)
